sdk/identity/identity/src/credentials/interactiveBrowserCredential.browser.ts

```diff
@@ -22,9 +21,8 @@ const logger = credentialLogger("InteractiveBrowserCredential");
  *
  * On NodeJS, it will open a browser window while it listens for a redirect response from the authentication service.
  *
- * On browsers, by default it uses the [Authorization Code Flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow)
- * and authenticates via popups. The `loginStyle` can be configured to use `redirect` instead, and the authentication flow can be configured to use the [Implicit Grant Flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-implicit-grant-flow)
- * by specifying the option `flow` with the value `implicit-grant`.
+ * On browsers, this credential uses the [Authorization Code Flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow)
```

Comment:
Isnt auth flow used by node as well? Right now when reading the above it looks like we are saying this applies to only browsers

Reply:
Perhaps we can move  the note on auth flow a few lines above before we start talking about node and browser?

Reply:
That is true! Thank you for the feedback 👍